### PR TITLE
Player and Skin upgrade

### PR DESCRIPTION
**Pull request by naeem91**
This PR contains following changes:

- Upgraded Ooyala core player  from v4.8.5 to 4.20.8
- Skin files were generated from forked version of HTMLSkin: https://github.com/mckinseyacademy/html5-skin/tree/development
- Skin config file old retrieval URL changed

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @naeem91! I've created [OSPR-2296](https://openedx.atlassian.net/browse/OSPR-2296) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



If you like, you can add yourself to the [AUTHORS](https://github.com/edx-solutions/xblock-ooyala/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

File: ooyala_player/ooyala_player.py [hunk 1]
```diff
@@ -170,14 +149,17 @@ def student_view(self, context):
         """
         Player view, displayed to the student
         """
-        # For lightchild xblock, pass skin resource URL, otherwise use our custom handler method
+        # Skin file path according to block type
```
Comment (msaqib52) [inline ooyala_player/ooyala_player.py]: if skin file is same for both block types than i think we can remove this if block

**Comment by nedbat** [comment]
@naeem91 Are you covered by the Arbisoft contributor agreement?

**Comment by naeem91** [comment]
@nedbat I've submitted the individual  contributor agreement.

**Comment by nedbat** [comment]
@naeem91 Thanks yes, I have that in my records, but are you an Arbisoft employee?

**Comment by naeem91** [comment]
@nedbat Yes. Although my agreement was submitted individually

**Comment by nedbat** [comment]
@naeem91 The OSPR bot thinks you are an outside contributor because of that.  I am going to switch you to the Arbisoft agreement so that we don't get this confusion.  Is there a reason to not switch you to Arbisoft?

**Comment by naeem91** [comment]
@nedbat I think no reason, you may make that switch. Thank You.

**Pull request merged**